spec/controllers/idv/in_person/ssn_controller_spec.rb

```diff
@@ -156,14 +169,18 @@
         expect(flow_session[:pii_from_user][:ssn]).to eq(ssn)
       end
 
+      it 'adds ssn to idv_session' do
+        put :update, params: params
+
+        expect(subject.idv_session.ssn).to eq(ssn)
+      end
+
       it 'invalidates steps after ssn' do
+        subject.idv_session.applicant = Idp::Constants::MOCK_IDV_APPLICANT
+
         put :update, params: params
 
         expect(subject.idv_session.applicant).to be_blank
-        expect(subject.idv_session.resolution_successful).to be_blank
-        expect(subject.idv_session.profile_confirmation).to be_blank
-        expect(subject.idv_session.vendor_phone_confirmation).to be_blank
-        expect(subject.idv_session.user_phone_confirmation).to be_blank
```

Comment:
These attributes are deleted from the spec because they have to be blank because they are set later by the phone step or review step, and ssn step can't be accessed when they are set.